lib/SPIRV/OCLUtil.cpp

```diff
@@ -238,16 +255,17 @@ transVecTypeHint(MDNode* Node) {
 SPIRAddressSpace
 getOCLOpaqueTypeAddrSpace(Op OpCode) {
   switch (OpCode) {
-  case OpTypePipe:
```

Comment:
According to SPIR 2.0.
sampler_t is defined as i32, so addr. space notion is not applicable.
image types and pipe qualified types are located in global memory.
So, I didn't include them here but I don't mind about images and pipes...